front/app/containers/Admin/users/GroupsListPanel.tsx

```diff
@@ -208,18 +198,18 @@ export const GroupsListPanel = ({
         <GroupName>
           <FormattedMessage {...messages.allUsers} />
         </GroupName>
-        <MembersCount>{usercount.count}</MembersCount>
+        <MembersCount>{usersCount?.data.attributes.count}</MembersCount>
       </MenuLink>
       <MenuLink to="/admin/users/admins-managers">
         <GroupName>
           <FormattedMessage {...messages.adminsAndManagers} />
         </GroupName>
-        {usercount.administrators_count !== null &&
-          usercount.moderators_count !== null && (
-            <MembersCount data-cy="e2e-admin-and-moderator-count">
-              {usercount.administrators_count + usercount.moderators_count}
-            </MembersCount>
-          )}
+        {usersCount && (
+          <MembersCount data-cy="e2e-admin-and-moderator-count">
```

Comment:
@IvaKop Is there a chance this e2e test `cypress/e2e/admin/users/seat-billing.cy.ts:81:13` started to fail because of this change? https://app.circleci.com/pipelines/github/CitizenLabDotCo/citizenlab/111147/workflows/5192b2dd-0459-46a4-bf4f-edba251dc440/jobs/262461